src/website/views/components/alert/_examples.pug

```diff
@@ -1699,3 +1734,16 @@ script.
     function() {
       chi.tab(document.querySelectorAll('.chi-tabs-panel .chi-tabs'));
     });
+    setupInteractiveAlert();
```

Comment:
I would change a little bit the `script` part:

![image](https://github.com/CenturyLink/Chi/assets/86958774/54e043ed-c792-4b08-b705-d33d121815bb)